first_objective_of_M2DVisualization.py

```diff
@@ -0,0 +1,30 @@
+'''This is a code to realize the first project of M2D visualization, running a simulation with logging enabled with only one packet.'''
+from tardis import run_tardis
+from tardis.io.config_reader import Configuration
+from tardis.io.atom_data.util import download_atom_data
+
+# Downloading atom data
+download_atom_data('kurucz_cd23_chianti_H_He')
+
+# Reading the Configuration stored in `tardis_config_packet_tracking.yml` into␣ ↪config
+config = Configuration.from_yaml("tardis_example.yml")
```

Comment:
Don't add this file on root directory if you haven't modified it. The example config already exists in `docs/` dir - use that